packages/version/src/version-command.ts

```diff
@@ -603,26 +614,46 @@ export class VersionCommand extends Command<VersionCommandOption> {
       })
     );
 
-    chain = chain.then(() =>
-      // update modern lockfile (version 2 or higher) when exist in the project root
-      loadPackageLockFileWhenExists(rootPath)
-        .then(lockFileResponse => {
-          if (lockFileResponse && lockFileResponse.lockfileVersion >= 2) {
-            for (const pkg of this.packagesToVersion) {
-              this.logger.silly(`lock`, `updating root "package-lock-json" for package "${pkg.name}"`);
-              updateTempModernLockfileVersion(pkg, lockFileResponse.json);
+    // update the project root npm lock file, we will read and write back to the lock file
+    // this is currently the default update and if none of the flag are enabled (or all undefined) then we'll consider this as enabled
+    if (npmClient === 'npm' && (this.options.manuallyUpdateRootLockfile || (this.options.manuallyUpdateRootLockfile === undefined && !this.options.syncWorkspaceLock))) {
+      this.logger.warn('npm', 'we recommend using --sync-workspace-lock which will sync your lock file via your favorite npm client instead of relying on Lerna-Lite itself to update it.');
+
+      chain = chain.then(() =>
+        // update modern lockfile (version 2 or higher) when exist in the project root
+        loadPackageLockFileWhenExists(rootPath)
+          .then(lockFileResponse => {
+            if (lockFileResponse && lockFileResponse.lockfileVersion >= 2) {
+              this.logger.verbose(`lock`, `start process loop of manually updating npm lock file`);
+
+              for (const pkg of this.packagesToVersion) {
+                this.logger.verbose(`lock`, `updating root "package-lock-json" for package "${pkg.name}"`);
+                updateTempModernLockfileVersion(pkg, lockFileResponse.json);
+              }
+
+              // save the lockfile, only once, after all package versions were updated
+              return saveUpdatedLockJsonFile(lockFileResponse.path, lockFileResponse.json)
+                .then((lockfilePath) => {
+                  if (lockfilePath) {
+                    changedFiles.add(lockfilePath);
+                  }
+                });
             }
+          })
+      );
+    }
 
-            // save the lockfile, only once, after all package versions were updated
-            return saveUpdatedLockJsonFile(lockFileResponse.path, lockFileResponse.json)
-              .then((lockfilePath) => {
-                if (lockfilePath) {
-                  changedFiles.add(lockfilePath);
-                }
-              });
-          }
-        })
-    );
+    // update lock file, with npm client defined when `--package-lock-only` is enabled
+    if (this.options.syncWorkspaceLock) {
```

Comment:
I'd put this in the else branch of https://github.com/ghiscoding/lerna-lite/pull/199/files#diff-dce25c7083d885ad853224dab7020a62243a4c81d90ad681a3fad3f50b2adaa4R620

Reply:
no I specifically target the ones who use npm and falls under that other flag, I don't want to annoy other users with this message

Reply:
I mean

```ts
if (npmClient === 'npm' && this.options.manuallyUpdateRootLockfile) {
 // ...
} else if (this.options.syncWorkspaceLock) {
 // ...
}
```